api/src/main/java/org/openmrs/module/patientqueueing/QueueingUtils.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * <p>
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.patientqueueing;
+
+import org.openmrs.api.AdministrationService;
+import org.openmrs.api.context.Context;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class QueueingUtils {
+	
+	private QueueingUtils() {
+	}
+	
+	/**
+	 * This method changes the time of a date set to the time given in the time string.
+	 * The Time format should be HH:mm:ss forexample 00:00:00 ,23:59:59 etc
+	 * 
+	 * @param date This is the date to be converted into set date format
+	 * @param time The time String which is appended to the date in-case it is not null
+	 * @return Date which is converted to required format
+	 * @throws ParseException
+	 */
+	public static Date changeTimeOfDate(Date date, String time) throws ParseException {
```

Comment:
Why do we change the time of the given date from what it already has?

Reply:
@dkayiwa if there is a senario  where you need to retrieved a record between two dates. such a functions comes in handy when it comes to changing time on date.

Reply:
I do not seem to get it. And when you say that such a functions come in handy, it looks like you do not have any current use case for it.